doc/documentation.rst

```diff
@@ -9,7 +9,7 @@ Documentation
    :Release: |version|
    :Date: |today|
 
-Previous versions:  `0.13.1 <http://nipype.readthedocs.io/en/0.13.1/>`_ `0.12.1 <http://nipype.readthedocs.io/en/0.12.1/>`_
+Previous versions: `0.14.0 <http://nipype.readthedocs.io/en/v0.14.0/>`_ `0.13.1 <http://nipype.readthedocs.io/en/0.13.1/>`_
```

Comment:
shouldn't this be 0.14.1

Reply:
0.14.1 became 1.0.

Reply:
:) i thought we had released 0.14.1 !! now i can understand the slowness of releases.